kit/Kit.cpp

```diff
@@ -2962,12 +2974,13 @@ bool globalPreinit(const std::string &loTemplate)
         }
     }
 
-    LokHookPreInit* preInit = reinterpret_cast<LokHookPreInit *>(dlsym(handle, "lok_preinit"));
+    LokHookPreInit2* preInit = reinterpret_cast<LokHookPreInit2 *>(dlsym(handle, "lok_preinit_2"));
     if (!preInit)
     {
-        LOG_FTL("No lok_preinit symbol in " << loadedLibrary << ": " << dlerror());
+        LOG_FTL("No lok_preinit_2 symbol in " << loadedLibrary << ": " << dlerror());
```

Comment:
Hmm - seems like the preinit2 thing looks sensible. I would suggest though that we add a new C wrapper to the LibreOfficeKitInit.h - or somesuch that uses it (?) bit of a pain changing these methods. I might also be inclined if we are fiddling with the signature there to add some const char *options parameter or something for future use - to avoid the next inevitable lok_preinit_3 biting us too soon ;-)